distributor-api-v1/operations.md

```diff
@@ -760,34 +760,60 @@ Gives a pricing information for the given configuration.
-| `Session` | number | required | Session number |
-| `LanguageCode` | string | required | Language code |
-| `CultureCode` | string | required | Culture Code |
+| `Client` | string | required | Identification of the client as described in [Authorization](https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/authorization). |
+| `HotelId` | string | required | Unique identifier of hotel. |
+| `Session` | number | required | Session number. |
+| `LanguageCode` | string | required | Language code. |
+| `CultureCode` | string | required | Culture Code. |
 
-### Response   <a id="response-5"></a>
+### Response  <a id="response-getpaymentconfiguration"></a>
 
 ```javascript
 {
-    {
-        "PaymentGatewayType": "Adyen",
-        "IsMerchant": false,
+    "PaymentGateway": {
+        "PaymentGatewayType": "PciProxy",
+        "PaymentCardStorageType": "PciProxy",
+        "IsMerchant": true,
         "SupportedCreditCardTypes": [
             "MasterCard",
             "Visa"
         ],
-        "PublicKey": "..."
+        "PublicKey": "1100016614",
+        "DefaultCurrencyCode": "EUR"
+    },
+    "SurchargeConfiguration": {
+        "SurchargeServiceId": "74d5eb0a-784a-4870-b34a-ab6500a1136e",
+        "SurchargeFees": {
+            "MasterCard": 0.02,
+            "Visa": 0.01,
+            "Amex": 0.0125
+        }
     }
 }
 ```
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
-| PaymentGatewayType | string | required | Type of the payment gateway \(`Adyen`, `Stripe` or `PciProxy`\) |
-| IsMerchant | boolean | required | Whether the gateway is processed through Mews Merchant or not |
-| SupportedCreditCardTypes | array of strings | required | The list of supported credit cards, should be used to enhance UX |
-| PublicKey | string | required | A key used by the payment gateway's client side library for authentication |
+| `PaymentGateway` | [PaymentGateway](operations.md#payment-gateway) | required | Object that describes payment gateway of the enterprise. |
+| `SurchargeConfiguration` | [SurchargeConfiguration](operations.md#surcharge-configuration) | required | Object describing surcharge configuration used by the enterprise. |
+
+#### SurchargeConfiguration <a id="surcharge-configuration"></a>
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `SurchargeServiceId` | string | optional | Unique identifier of surcharge service, if enterprise does not use surcharge feature this Id will be `null`. |
+| `SurchargeFees` | collection of key-value pairs where key is string and value is number | optional | Collection of key-value pairs where key is string representation of [CreditCardType](operations.md#creditcardtype) and value is amount of the surcharge fee itself. If enterprise does not use surcharge feature this will be an empty object. |
```

Comment:
@dobras What do you think about the change from `dictionary` to `key-value pairs`? I thought about it a bit more and maybe it would actually be better if we just write `object` as @zdrazil suggested, so here it would be e.g. `object where each key is string and value is number`. What do you think? Or do you have any better suggestions? Using `collection` does sound a bit weird in JSON.

Reply:
Im not an expert on json but this just looks like an object that has properties where name of the property is a string and value is number. And it has a lot of them. It does not have `][` these brackets to indicate collection.